docs/source/developer_guide/performance/optimization_and_tuning.md

```diff
@@ -0,0 +1,207 @@
+# Optimization and Tuning
+
+## Motivation
+
+To achieve ultimate performance on vllm-asend `v0.7.3` with mindie-turbo `2.0rc1`, we have made efforts to optimize our compilation, environment variables, application configs, etc.
```

Comment:
This guide aims to help users to improve vllm-ascend performance on system level. It includes OS configuration, library optimization, deploy guide and so on. Any feedback is welcome.